### PR TITLE
Update Blog Post “studio-launch”

### PR DESCRIPTION
**Pull request by ksatx**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit e3d54375b9a50da6ae826df457e364fab2946997

https://deploy-preview-1250--stoplightio.netlify.com

File: netlify/blog-posts/studio-launch.md [hunk 1]
```diff
@@ -8,28 +8,26 @@ tags:
...
-publishedDate: 2019-08-21
+publishedDate: 2019-08-21T00:00:00.000Z
 author: Marc MacLeod
-title: Announcing Stoplight Studio, Our Next Gen API Designer
-subtitle: Written from scratch with the modern development workflow in mind. Now with OpenAPI v3 support, and options to use your own version control system.
-listImage: /images/studio/blog-list.png
+title: 'Announcing Stoplight Studio, Our Next Gen API Designer'
+subtitle: >-
+  Written from scratch with the modern development workflow in mind. Now with
+  OpenAPI v3 support, and options to use your own version control system.
```
Comment (rowa97) [inline netlify/blog-posts/studio-launch.md]: need to re-add a listImage

Comment (ksatx) [reply netlify/blog-posts/studio-launch.md]: re-added. Not sure where it went in the first place ¯\_(ツ)_/¯

**Comment by rowa97** [comment]
lets keep an eye on this after publishing because in preview the image still isn't showing up but that might just be because its an outdated preview

**Pull request merged**